ASF-slaves.txt

```diff
@@ -27,12 +27,41 @@ Add ssh pub key from puppet deployment repo:
   $ sudo chmod 700 /home/jenkins/.ssh
   $ sudo chmod 600 /home/jenkins/.ssh/authorized_keys
 
+Install and Configure Puppet:
+    1. I added an 'asf999' user which has a pub key file containing keys from the Infra Team. (It has our standard uid/gid of 910)
+        (See also: https://github.com/apache/infrastructure-p6/blob/production/data/common/asf999.yaml )
+    2. The asf999 user has sudo no passwd (configured in visudo)
+    3. Recreated the Jenkins user account, see more below (in Option A)
+    4. Installed Puppet 6  (Not Puppet 5 that Bionic would normally install)
+         and configured the puppet.conf file ready for use
+      4a. wget https://apt.puppetlabs.com/puppet-release-bionic.deb
+      4b. dpkg -i puppet-release-bionic.deb
+      4c. apt-get update
+      4d. apt-get install -y puppet-agent
+      4e. cat >> /etc/puppetlabs/puppet/puppet.conf << EOF
+        [main]
+        server = pm01-lw-us.apache.org
+        EOF
+
+
+
+Ensure `hostname` is configured to the server's public ip.
+
+Ensure the additional data volume is mounted at `/home/jenkins`
```

Comment:
It's a 500 GB data volume.  This is called out in some other instructions list, might be nice to have it all neatly in one place.